docs/content/preview/explore/change-data-capture/cdc-tutorials/cdc-aws-msk.md

```diff
@@ -0,0 +1,240 @@
+        "kafkaconnect:ListCustomPlugins",
+        "cloudwatch:DeleteMetricStream",
+        "cloudwatch:SetAlarmState",
+        "kafka-cluster:DescribeGroup",
+        "cloudwatch:GetMetricWidgetImage",
+        "kafkaconnect:DescribeCustomPlugin",
+        "s3:*",
+        "kafka-cluster:DescribeCluster",
+        "cloudwatch:EnableInsightRules",
+        "cloudwatch:PutCompositeAlarm",
+        "cloudwatch:PutMetricStream",
+        "cloudwatch:PutInsightRule",
+        "cloudwatch:PutMetricAlarm",
+        "cloudwatch:EnableAlarmActions",
+        "cloudwatch:ListMetricStreams",
+        "kafkaconnect:CreateCustomPlugin",
+        "kafkaconnect:DeleteConnector",
+        "kafkaconnect:DeleteCustomPlugin",
+        "kafka-cluster:WriteData"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "VisualEditor5",
+      "Effect": "Allow",
+      "Action": "ec2:DescribeNetworkInterfaces",
+      "Resource": "arn:aws:ec2:*:*:network-interface/*",
+      "Condition": {
+        "StringEquals": {
+          "ec2:ResourceTag/AmazonMSKConnectManaged": "true"
+        }
+      }
+    }
+  ]
+}
+```
+
+### Enable CDC on YugabyteDB
+
+Ensure that YugabyteDB is up and running. To install YugabyteDB on your cloud virtual machine, refer to [Quick start](../../../../quick-start/linux/).
+
+Create a test table:
+
+```sql
+CREATE TABLE test (id INT PRIMARY KEY, name TEXT);
+```
+
+Enable CDC using yb-admin. The following command enables CDC on all the schemas and tables in the YugabyteDB database.
+
+```sh
+./bin/yb-admin — master_addresses <master_addresses>:7100 create_change_data_stream ysql.yugabyte
+```
+
+If you have a multi-node YugabyteDB setup, you need to provide a comma-separated list of **host:port** values of both the leader and the follower nodes as the `master_addresses` argument.
+
+If successful, the command returns the CDC stream ID:
+
+```output
+CDC Stream ID: 90fe97d59a504bb6acbfd6a940
+```
+
+For more information on CDC commands, refer to [Change data capture commands](../../../../admin/yb-admin/#change-data-capture-cdc-commands).
+
+### Configure the AWS Security Group
+
+Create a Security Group with inbound and outbound rules configured to ensure access to the MSK cluster and YugabyteDB. In this example, enable incoming traffic from all the ports.
+
+![Edit Inbound Rules](/images/explore/cdc/aws_msk_images/edit-inbound-rules.png)
+
+### Upload Debezium connector Jar file to the S3 bucket
+
+Download the YugabyteDB Debezium connector jar from the [repository](https://github.com/yugabyte/debezium-connector-yugabytedb/releases/download/v1.9.5.y.19/debezium-connector-yugabytedb-1.9.5.y.19.jar) and upload it to an S3 bucket.
+
+![Upload to S3](/images/explore/cdc/aws_msk_images/upload-to-s3.png)
+
+### Configure the Amazon MSK cluster
+
+This example creates an Amazon MSK cluster in same VPC as that of the YugabyteDB cluster. Note that this is a generic configuration, it might differ based your organizational IT policy.
```

Comment:
```suggestion
For this example, create an Amazon MSK cluster in the same VPC as that of the YugabyteDB cluster. Note that this is a generic configuration, and it may differ based on your organizational IT policy.
```